agent/core/src/main/java/org/jolokia/restrictor/RestrictorFactory.java

```diff
@@ -45,18 +71,19 @@ public static Restrictor createRestrictor(Configuration pConfig, LogHandler logH
             return customRestrictor;
         }
 
-        String location = NetworkUtil.replaceExpression(pConfig.get(ConfigKey.POLICY_LOCATION));
+        String rawLocation = pConfig.get(ConfigKey.POLICY_LOCATION);
+        String location = NetworkUtil.replaceExpression(rawLocation);
         try {
             Restrictor ret = RestrictorFactory.lookupPolicyRestrictor(location);
             if (ret != null) {
-                logHandler.info("Using policy access restrictor " + location);
+                logHandler.info("Using policy access restrictor " + rawLocation);
```

Comment:
Any particular reason why you dont want to log the resolved location, but the specified one ?